desktop_version/src/Script.cpp

```diff
@@ -268,25 +268,49 @@ void scriptclass::run(void)
             }
             if (words[0] == "destroy")
             {
-                if(words[1]=="gravitylines"){
-                    for(size_t edi=0; edi<obj.entities.size(); edi++){
-                        if(obj.entities[edi].type==9) obj.disableentity(edi);
-                        if(obj.entities[edi].type==10) obj.disableentity(edi);
+                if (words[1] == "gravitylines")
+                {
+                    for (size_t edi = 0; edi < obj.entities.size(); edi++)
+                    {
+                        if (obj.entities[edi].type == 9 || obj.entities[edi].type == 10)
+                        {
+                            obj.disableentity(edi);
+                        }
                     }
-                }else if(words[1]=="warptokens"){
-                    for(size_t edi=0; edi<obj.entities.size(); edi++){
-                        if(obj.entities[edi].type==11) obj.disableentity(edi);
+                }
+                else if (words[1] == "warptokens")
+                {
+                    for (size_t edi = 0; edi < obj.entities.size(); edi++)
+                    {
+                        if (obj.entities[edi].type == 11)
+                        {
+                            obj.disableentity(edi);
+                        }
                     }
-                }else if(words[1]=="platforms"||words[1]=="moving"){
-                    bool fixed=words[1]=="moving";
-                    for(size_t edi=0; edi<obj.entities.size(); edi++){
-                        if(fixed) obj.disableblockat(obj.entities[edi].xp, obj.entities[edi].yp);
-                        if(obj.entities[edi].rule==2 && obj.entities[edi].animate==100) obj.disableentity(edi);
+                }
+                else if (words[1] == "platforms" || words[1] == "moving")
+                {
+                    for (size_t edi = 0; edi < obj.entities.size(); edi++)
+                    {
+                        if (obj.entities[edi].rule == 2 && obj.entities[edi].animate == 100)
+                        {
+                            if (words[1] == "moving")
```

Comment:
Please keep the `bool`, as an explicit optimization in case the compiler can't see it, and as self-documenting code so readers know that `moving` is specifically the fixed version of `platforms`.
```suggestion
                    const bool fixed = words[1] == "moving";

                    for (size_t edi = 0; edi < obj.entities.size(); edi++)
                    {
                        if (obj.entities[edi].rule == 2 && obj.entities[edi].animate == 100)
                        {
                            if (fixed)
```